### PR TITLE
Doc 323 redesign runs draft

### PR DESCRIPTION
**Pull request by oojo12**
# What does this PR do?

Redesigns the runs content to make it flow as a user journey. The thought is that this type of organization will make the Runs documentation more accessible. It is known that some of the links will not work. This is due to the desired convention of linking to the official docs site and because one of the links are going to come from the update GitHub docs PR.

Contains the following and more:
1. naive tutorials as an introduction to runs 
2. links to FAQ page at the top of all documentation content
3. Top level pages with links to subpages
4. Flowing like a general user story through the product

### CODE REVIEW

**Comment by oojo12** [comment]
@essiequoi you are welcome to directly commit the desired changes to the files. In cases where you have questions please leave a comment and I can address them.

**Comment by essiequoi** [comment]
@oojo12 if you use internal links (../page.md) instead of external ones, we'll be able to preview and make sure navigation works. 
also i'm unable to build this locally, there are errors because of sidebar.js did you run into that?

**Comment by oojo12** [comment]
@essiequoi I am aware but I am not going to change it to internal links just to go back to external links. Also the intention was for sidebar.js fix to be merged and then this to be rebased on that. If you want to review you will have to use GitHub and check the pages + the links manually.

File: docs/features/runs/overview.md [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
-# ⚡Runs
+# Demystifying Runs
+
+## What are Runs
+Runs are "serverless" compute dedicated to executing the scripts you provide them.  
+
+## Why use Runs
+Because Runs are "serverless" compute you only pay for the time your script is running. This amounts to massive cost savings whether you use Runs in Grid Cloud or in Bring Your Own Cloud.
+
+### Capabilities Highlights
```
Comment (oojo12) [inline docs/features/runs/overview.md:11]: I would love product eyes on the capabilities we highlight.

**Review by essiequoi — APPROVED** [review]

**Comment by essiequoi** [comment]
@oojo12 @pritamsoni-hsr need to resolve merge conflicts

**Comment by oojo12** [comment]
Cancelling since one has been opened against dev instead.

**Comment by oojo12** [comment]
I'm aware. This will be merged into the development branch instead of
master. The bugbash should be done off development and development made a
formal release after and merged into master.

**Pull request closed without merging**